File: google-function-auth/function_code/handler.js
// --- Your app logic goes here ---
// This is the only file you change per project.
// It receives the authenticated request and sends the response.

function handle(req, res) {
  const name = req.body?.name || "World";
  res.json({ message: `Hello, ${name}!` });
}

module.exports = { handle };

File: google-function-auth/function_code/handler.py
from flask import jsonify


def handle(request):
    """Your app logic goes here.
    This is the only file you change per project.
    """
    data = request.get_json(silent=True) or {}
    name = data.get("name", "World")
    return jsonify({"message": f"Hello, {name}!"})
